modules/ROOT/pages/guided-setup.adoc

```diff
@@ -8,60 +8,51 @@ From the Anypoint API Community Manager control panel, open the Salesforce App L
+. If your Anypoint user does not have organization administrator credentials:
+.. Ask your organization administrator to use the following steps to create a connected app and give you the *Client ID* and *IDP ID*.
+.. Select *I don't have organization administrator credentials* and enter the *Client ID* and *IDP ID* from your organization administrator.
+
+To create a connected app if you are an organization administrator:
+
+. Create a key pair in Salesforce for communication between Salesforce and Anypoint Platform:
+.. Click *Setup* > *Security* > *Certificate and Key Management* > *Create Self-Signed Certificate*.
+.. Set *Label/Name* to `Anypoint`.
+.. Disable *Exportable Private Key*.
+.. Set *Key Size* to 2048 or larger.
+.. Click *Save*.
+.. Click *Download Certificate* to save the public key on your local machine.
+. Click *Access Management* > *Connected Apps* > *Create App*.
+. Enter a name for the app.
+. Enable *App acts on behalf of a user*.
+. In the section *Grant types*, enable *JWT Bearer*.
+. Copy the public key from the downloaded file and paste it into the public key text area.
+. Verify that the public key certificate has this format:
 +
 ----
 -----BEGIN CERTIFICATE-----
 PUBLIC KEY
 -----END CERTIFICATE-----
 ----
-.. Set *Website URL* to `+https://login.salesforce.com+`.
-.. In the section *Redirect URIs*, add `+http://localhost+`, because connected apps require redirect URIs but this configuration does not use them.
-.. In the section *Who can use this application?*, select *Members of this organization only*.
-.. In the section *Scopes*, add *Background Access* and *Full Access*.
-.. Click *Save*.
-.. Copy the client ID.
-.. Do not copy the client secret, because this configuration uses the certificate instead.
-. Authenticate and authorize the connected app to access your Anypoint organization.
-+
-If your Anypoint account has the organization administrator role, provide your Anypoint Platform user and password, and the guided setup will create the app.
-+
-Otherwise, provide these app details:
-+
-.. Set *Client ID* to the client ID from the connected app you created.
-.. To find the identity provider ID, run this request:
+. Set *Website URL* to `+https://login.salesforce.com+`.
+. In the section *Redirect URIs*, add `+http://localhost+`, because connected apps require redirect URIs but this configuration does not use them.
+. In the section *Who can use this application?*, select *Members of this organization only*.
+. In the section *Scopes*, add *Background Access* and *Full Access*.
+. Click *Save*.
+. Copy the client ID.
+. Do not copy the client secret, because this configuration uses the certificate instead.
```

Comment:
Not a step; reformat as follow-on para to previous step.

Reply:
Fixed.